.gitignore

```diff
@@ -22,6 +22,8 @@ py3[0-9]_native/
 a.out
 src/python/grpcio_*/LICENSE
 src/python/grpcio_status/grpc_status/google/rpc/status.proto
+None
```

Comment:
`None`? What generates this?

Reply:
I'm not entire sure which line of code produces it. Building from `setup.py` will generate a folder named `None`. I presume some Python script has gone wrong. I dropped this line of ignore, so in future if it appears again we may investigate.